eyieko/landville-frontend
# #165880952 view similar properties

## Description
A user should see the properties that look like the current ones they are viewing. They have the option of viewing properties:
- In the same location
- In the same price range
- With the same number of bedrooms
- With the same number of bathrooms

## Type of change


## How Has This Been Tested?


## Checklist:


## How to test this PR
 1. Clone the repo and checkout to this branch `ft-view-similar-properties-165880952`
 2. Start the server 
 3. Visit `properties` -> click on a property -> scroll down to `See other properties` and select an option

## Related Pivotal Tracker stories
[#165880952](https://www.pivotaltracker.com/n/projects/2343772/stories/165880952)
